test/helpers.js

```diff
@@ -1,9 +1,5 @@
-function shouldWarn(about) {
+export function shouldWarn(about) {
   console.warn.called.should.be.true;
   console.warn.calledWithMatch(about).should.be.true;
   console.warn.reset();
 }
-
-export default {
```

Comment:
That's my fault.
At some middle point of review process there was proposed version with additional exported function.
I've moved that function out, but forgot to simplify this.
:+1: